AuroraEditor/Base/AppPreferences/UI/ThemePreferences/EditorThemeView.swift

```diff
@@ -16,74 +16,75 @@ struct HighlightThemeView: View {
+//                        }
+//                        .frame(maxWidth: .infinity, alignment: .leading)
+//                    }
+//                    .padding(.bottom, 20)
+//                    Text("Syntax")
+//                        .font(.system(size: 13, weight: .semibold))
+//                        .foregroundColor(.secondary)
+//                        .padding(.bottom, 10)
+//                    HStack(spacing: 0) {
+//                        VStack(alignment: .leading, spacing: 10) {
+//                            PreferencesColorPicker($themeModel.themes[index].editor.keywords.swiftColor,
+//                                                   label: "Keywords")
+//                            PreferencesColorPicker($themeModel.themes[index].editor.commands.swiftColor,
+//                                                   label: "Commands")
+//                            PreferencesColorPicker($themeModel.themes[index].editor.types.swiftColor,
+//                                                   label: "Types")
+//                            PreferencesColorPicker($themeModel.themes[index].editor.attributes.swiftColor,
+//                                                   label: "Attributes")
+//                            PreferencesColorPicker($themeModel.themes[index].editor.variables.swiftColor,
+//                                                   label: "Variables")
+//                        }
+//                        .frame(maxWidth: .infinity, alignment: .leading)
+//                        VStack(alignment: .leading, spacing: 10) {
+//                            PreferencesColorPicker($themeModel.themes[index].editor.values.swiftColor,
+//                                                   label: "Values")
+//                            PreferencesColorPicker($themeModel.themes[index].editor.numbers.swiftColor,
+//                                                   label: "Numbers")
+//                            PreferencesColorPicker($themeModel.themes[index].editor.strings.swiftColor,
+//                                                   label: "Strings")
+//                            PreferencesColorPicker($themeModel.themes[index].editor.characters.swiftColor,
+//                                                   label: "Characters")
+//                            PreferencesColorPicker($themeModel.themes[index].editor.comments.swiftColor,
+//                                                   label: "Comments")
+//                        }
+//                        .frame(maxWidth: .infinity, alignment: .leading)
+//                    }
+//                }
+//                .padding(20)
+//            } else {
+//                Text("Select a Theme")
+//                    .foregroundColor(.secondary)
+//                    .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .center)
+//            }
+//        }
+        EmptyView()
```

Comment:
Do you need the commented code for reference?

Reply:
tbh we may be going a different approach since we have more options for themes (beyond simple colour changes)

Reply:
yeah ill remove it